README.rst

```diff
@@ -44,11 +44,585 @@ Available Features
 Dependencies
 ============
 
```

Comment:
\imo it's not really a "feature" of the application that concerns user when they are reading the README to understand what the project is accomplishing. I would consider infinite scroll as a UI component instead and a better way to show UI components is images / video / gifs! :smile:

But if you want to add, let me know! :smile:

Reply:
yes you're right, no need to mention the infinite scroller, we should just make sure it's clear that users can easily and quickly browse the address space.
I would still like to show the infinite scroller in a gif because I think it's a cool feature that may result useful to some.

Reply:
Done! :smile: